database/mysql/mysql_user.py

```diff
@@ -307,19 +317,28 @@ def privileges_revoke(cursor, user,host,db_table,grant_option):
     query = ["REVOKE ALL PRIVILEGES ON %s" % mysql_quote_identifier(db_table, 'table')]
     query.append("FROM %s@%s")
     query = ' '.join(query)
-    cursor.execute(query, (user, host))
+    try:
+        cursor.execute(query, (user, host))
+    except Exception, e:
+        raise Exception("%s. Query=\"%s\"" % (str(e), query % (user, host)))
```

Comment:
The exceptions that we throw here we want to catch later when we use module.fail_json() (in the main function).  So don't raise Exception, instead either raise one of the Exceptions we already catch or create a new exception class and add that as something that we catch there.